plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -149,7 +149,7 @@
          on to the appropriate Host (virtual host).
          Documentation at /docs/config/engine.html -->
 
-    <Engine name="Catalina" defaultHost="localhost">
+    <Engine name="Catalina" defaultHost="localhost" jvmRoute="${exo.cluster.node.name}">
```

Comment:
> Yep, but more manual configuration stuff to care about
> Our goal is to make the configuration of cluster as much simple as possible
Yes exactly, you touched the point, this variable is not necessary to make load balancing as the load balancers mostly used their own cookies or directly the session id to make to routing.
Changing the default behavior will add one more thing to worry about while it is not needed.